charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-cleanup-script.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-proxy-cleanup-script
+  namespace: kube-system
+  labels:
+    app: kubernetes
+    gardener.cloud/role: system-component
+    origin: gardener
+    role: proxy
+data:
+  cleanup.sh: |
+    #!/bin/sh -e
+    OLD_KUBE_PROXY_MODE="$(cat "$1")"
+    if [ -z "${OLD_KUBE_PROXY_MODE}" ] || [ "${OLD_KUBE_PROXY_MODE}" == "${KUBE_PROXY_MODE}" ]; then
+      echo "${KUBE_PROXY_MODE}" >"$1"
```

Comment:
```suggestion
      echo "${KUBE_PROXY_MODE}" >"$1"
      echo "there is nothing to cleanup, exiting"
```

Reply:
duplicate of https://github.com/gardener/gardener/pull/2238#discussion_r424193815

Reply:
Ops, Rafael also has a similar proposal above.